CMakeLists.txt

```diff
@@ -287,6 +287,9 @@ The %{name} package also contains HTML formatted documents containing
 hardening guidances that have been generated from XCCDF benchmarks.
 ")
 
+include(CTest)
+FILE(GLOB_RECURSE ALL_DATASTREAMS "ssg-*-ds.xml")
```

Comment:
code style
GLOB_RECURSE makes no sense, it's all in one directory. it also won't work because this is run at a time when the datastreams are not built, unless you configure multiple times this can't work

Reply:
what is it even used for?

Reply:
At the moment, the line can go away.
However, there was a test that operated on all built datastreams (that were saved to the variable). I thought that it may come in handy when some other tests are added to the suite, so I kept it there.
Anyway, when I have replaced `GLOB_RECURSE` by `GLOB`, it ceased to work. I agree that it doesn't make sense, but what to do if it is the only way how to make it work?

Reply:
Debug, it has to work with just GLOB. If it doesn't you are either doing something wrong (implicit path would be my bet) or there is a bug in cmake.

Reply:
> I thought that it may come in handy when some other tests are added to the suite, so I kept it there.

Please remove it as it is not relevant to this PR. In general I think this line of thinking can be valid if the code is really complex and took forever to figure out but in this case I think we should remove it, it's not used and would only cause confusion. In the future when we add ctest in SSG we can add "add_testing" again...

Reply:
OK, I will remove the CMake modification from the PR.